pkg/cloudprovider/gce/gce.go

```diff
@@ -241,7 +241,9 @@ func (gce *GCECloud) CreateTCPLoadBalancer(name, region string, externalIP net.I
 	if err != nil {
 		return nil, err
 	}
-	return net.ParseIP(fwd.IPAddress), nil
+	loadBalancerInfo := &api.LoadBalancerInfo{Rewrite: false}
```

Comment:
Because the provider is the only thing that can know what type of Load Balancer the cloud provider has - it's knowledge provided by the system, not by the user.